graylog2-web-interface/src/components/bootstrap/imports.js

```diff
@@ -19,7 +19,6 @@
 export {
   /* 👇 no custom theme colors needed 👇 */
   ButtonGroup,
-  ButtonToolbar,
```

Comment:
Creating a separate file for the `ButtonToolbar` was necessary to avoid a dependency cycle.